ui/pages/institutional/custody/custody.tsx

```diff
@@ -593,10 +685,9 @@ const CustodyPage = () => {
               id="selectAllAccounts"
               data-testid={`select-all-accounts-selected-${isCheckBoxSelected}`}
               name="selectAllAccounts"
-              marginLeft={2}
```

Comment:
is this valid to remove?

Reply:
Yes, it is. Antonio has checked at the storybook to see if makes sense to remove it